File: CPP/Program/Periferal.cpp
#ifndef PERIFERAL_CPP
#define PERIFERAL_CPP

#include <iostream>
#include <string>
using namespace std;

class Periferal {
private:
    string jenis;
    string koneksi; // USB, Bluetooth, Wireless, dll
    
public:
    Periferal() {
    }
    
    Periferal(string jenis, string koneksi) {
        this->jenis = jenis;
        this->koneksi = koneksi;
    }
    
    void setJenis(string jenis) {
        this->jenis = jenis;
    }
    
    void setKoneksi(string koneksi) {
        this->koneksi = koneksi;
    }
    
    string getJenis() {
        return this->jenis;
    }
    
    string getKoneksi() {
        return this->koneksi;
    }
    
    ~Periferal() {
    }
};

#endif // PERIFERAL_CPP

File: Python/Program/Periferal.py
class Periferal:
    def __init__(self, jenis="", koneksi=""):
        self.__jenis = jenis
        self.__koneksi = koneksi
    
    def set_jenis(self, jenis):
        self.__jenis = jenis
    
    def set_koneksi(self, koneksi):
        self.__koneksi = koneksi
    
    def get_jenis(self):
        return self.__jenis
    
    def get_koneksi(self):
        return self.__koneksi